src/index.html

```diff
@@ -5,8 +5,348 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a class="header__logo" href="#home"></a>
+      <nav class="header__navigation">
+        <ul class="header__navlist">
+          <li class="header__listitem">
+            <a href="#apple" class="header__link is-active">Apple</a>
+          </li>
+          <li class="header__listitem">
```

Comment:
Add empty lines between multiline sibling blocks of HTML. Please check for all cases!
```suggestion

          <li class="header__listitem">
```